packages/backend-proxy-middleware/test/base/proxy.test.ts

```diff
@@ -219,14 +219,33 @@ describe('proxy', () => {
             mockListDestinations.mockResolvedValueOnce({
                 [backend.destination]: {
                     Host: 'http://backend.example/sap',
-                    WebIDEUsage: `${WebIDEUsage.ODATA_GENERIC},${WebIDEUsage.FULL_URL}`
+                    WebIDEUsage: `${WebIDEUsage.ODATA_GENERIC}`,
+                    WebIDEAdditionalData: 'full_url'
```

Comment:
Why did you replace the constant here?

Reply:
Because there is no `full_url` value for the property `WebIDEUsage`. The idea was to not cause any confusion.

Reply:
Let me rephrase, shouldn't it be
```
[backend.destination]: {
                    Host: 'http://backend.example/sap',
                    WebIDEUsage: WebIDEUsage.ODATA_GENERIC,
                    WebIDEAdditionalData: WebIDEUsage.FULL_URL
                }
```

Reply:
Even cleaner, should we update the constants?

Reply:
Yes, we should update the constants. I already did some update here https://github.com/SAP/open-ux-tools/pull/609/files#diff-32ab4a9ed3098e009c7d23e32540229f4c16ead04750a87224d55d42acd4664fL8. I can introduce one new enum called `WebIDEAdditionalData`?

Reply:
A new enum would be my preference. Do we know more possible values? I couldn't find any doc.

Reply:
This is what I found: https://help.sap.com/docs/help/825270ffffe74d9f988a0f0066ad59f0/5c3debce758a470e8342161457fd6f70.html?locale=en-US

<img width="997" alt="image" src="https://user-images.githubusercontent.com/91968887/182380823-ed6d4ed2-48b1-4a3f-a8b6-93ce0b7f335c.png">

Reply:
Done with https://github.com/SAP/open-ux-tools/pull/609/commits/22a938b34e4d0153464b38bd2239261478153a98